TELCODOCS-1077: LVMS via ZTP (#58491)
openshift/openshift-docs: modules/ztp-sno-du-configuring-lvms.adoc

= {lvms}

You can dynamically provision local storage on {sno} clusters with {lvms-first}.

[NOTE]
====

> The recommended storage solution for {sno} is the Local Storage Operator. Alternatively, you can use {lvms} but it requires additional CPU resources to be allocated.

Reviewer: Just a suggestion. Up to you

```suggestion
The recommended storage solution for {sno} is the Local Storage Operator. Alternatively, you can use {lvms} which requires additional CPU resources to be allocated.
```
